src/dns/mod.rs

```diff
@@ -161,6 +161,17 @@ impl Resolver {
             timeout: Some(Duration::from_secs(3)),
         }
     }
+    
+    /// Creates a new resolver using the [Google Public DNS][cf] service.
+    ///
+    /// [cf]: https://developers.google.com/speed/public-dns/
```

Comment:
looks good! `cf` was used as an abbreviation for cloudflare, please replace it with something else. :)